src/Resources/config/services.xml

```diff
@@ -10,7 +10,7 @@
 
     <parameters>
         <parameter key="default_logo_file">@SyliusRefundPlugin/Resources/assets/sylius-logo.png</parameter>
-        <parameter key="sylius.refund.template.logo_file">%env(default:default_logo_file:resolve:SYLIUS_REFUND_LOGO_FILE)%</parameter>
+        <parameter key="sylius_refund.template.logo_file">%env(default:default_logo_file:resolve:SYLIUS_REFUND_LOGO_FILE)%</parameter>
```

Comment:
You shouldn't change the parameter name as it could be used by someone in the end application, you can add a new parameter and deprecate the old one.

Reply:
I wasn't sure how to accomplish it. For now, I reverted that change